src/components/common/Header/MobileHeader/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { cn } from '@bcsdlab/utils';
+import { useEffect } from 'react';
+import { useLocation, useNavigate, useParams } from 'react-router-dom';
+import { CATEGORY } from 'static/category';
+import useBooleanState from 'utils/hooks/useBooleanState';
+import useLogger from 'utils/hooks/useLogger';
+import useMediaQuery from 'utils/hooks/useMediaQuery';
+import * as api from 'api';
+import { createPortal } from 'react-dom';
+import { ReactComponent as HamburgerIcon } from 'assets/svg/hamburger-icon.svg';
+import { ReactComponent as KoinServiceLogo } from 'assets/svg/koin-service-logo.svg';
+import { ReactComponent as WhiteArrowBackIcon } from 'assets/svg/white-arrow-back-icon.svg';
+import styles from './MobileHeader.module.scss';
+import Panel from './Panel';
+
+function useMobileSidebar(pathname: string, isMobile: boolean) {
+  const [isExpanded, expandSidebar, hideSidebar] = useBooleanState(false);
```

Comment:
state간의 이름 통일이 필요해보여요 😅 
```suggestion
  const [isSidebarExpanded, setSidebarExpanded, toggleSidebarExpanded] = useBooleanState(false);
```

Reply:
수정하였습니다.